### PR TITLE
Flame: retimed attributes are integrated with settings

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Brief description
Fixing retiming attributes

## Description
Retimed framerange and handles were not integrated in publishing. Settings were not created for Creator to control retimed attributes.

### CODE REVIEW

File: openpype/hosts/flame/plugins/create/create_shot_clip.py [hunk 1]
```diff
@@ -23,10 +23,10 @@ def process(self):
                 # nested dictionary (only one level allowed
                 # for sections and dict)
                 for _k, _v in v["value"].items():
-                    if presets.get(_k):
+                    if presets.get(_k, None) is not None:
                         gui_inputs[k][
                             "value"][_k]["value"] = presets[_k]
-            if presets.get(k):
+            if presets.get(_k, None) is not None:
```
Comment (BigRoy) [inline openpype/hosts/flame/plugins/create/create_shot_clip.py:29]: Shouldn't this be this?
```suggestion
            if presets.get(k, None) is not None:
```

File: openpype/hosts/flame/plugins/publish/extract_subset_resources.py [hunk 2]
```diff
@@ -90,38 +90,38 @@ def process(self, instance):
         handle_end = instance.data["handleEnd"]
         handles = max(handle_start, handle_end)
         include_handles = instance.data.get("includeHandles")
-        retimed_handles = instance.data.get("retimedHandles")
+        not_retimed_handles = instance.data.get("notRetimedHandles")
```
Comment (BigRoy) [inline openpype/hosts/flame/plugins/publish/extract_subset_resources.py:93]: Out of interest - what's the reason this logic is inversed.
The setting is **Retime Handles** but we're storing and using **Not Retime Handles**?

I feel it's slightly adding to the complexity of the code by having that inverted.

File: openpype/hosts/flame/plugins/create/create_shot_clip.py (hunk 1 above)
Comment (jakubjezek001) [reply openpype/hosts/flame/plugins/create/create_shot_clip.py]: Very good catch. Thanks ;)

File: openpype/hosts/flame/plugins/publish/extract_subset_resources.py (hunk 2 above)
Comment (jakubjezek001) [reply openpype/hosts/flame/plugins/publish/extract_subset_resources.py:93]: Yes, I agree. The only reason is backwards compatible with all hosts. But perhaps there is a better way of doing this. I will change this.

Comment (jakubjezek001) [reply openpype/hosts/flame/plugins/publish/extract_subset_resources.py]: Check it now please if it makes more sense @BigRoy

File: openpype/hosts/flame/plugins/create/create_shot_clip.py [hunk 3]
```diff
@@ -23,10 +23,11 @@ def process(self):
                 # nested dictionary (only one level allowed
                 # for sections and dict)
                 for _k, _v in v["value"].items():
-                    if presets.get(_k):
+                    if presets.get(_k, None) is not None:
```
Comment (iLLiCiTiT) [inline openpype/hosts/flame/plugins/create/create_shot_clip.py:26]:
```suggestion
                    if presets.get(_k) is not None:
```

File: openpype/hosts/flame/plugins/create/create_shot_clip.py [hunk 4]
```diff
@@ -23,10 +23,11 @@ def process(self):
...
                         gui_inputs[k][
                             "value"][_k]["value"] = presets[_k]
-            if presets.get(k):
+
+            if presets.get(k, None) is not None:
```
Comment (iLLiCiTiT) [inline openpype/hosts/flame/plugins/create/create_shot_clip.py:30]:
```suggestion
            if presets.get(k) is not None:
```

File: openpype/hosts/flame/plugins/publish/extract_subset_resources.py [hunk 5]
```diff
@@ -159,14 +162,29 @@ def process(self, instance):
...
+            if not retimed_handles:
+                instance.data["versionData"].update({
+                    "handleStart": handle_start,
+                    "handleEnd": handle_end,
+                    "frameStart": vd_frame_start,
+                    "frameEnd": (
+                        (vd_frame_start + source_duration_handles - 1)
+                        - (handle_start + handle_end)
+                    )
+                })
```
Comment (iLLiCiTiT) [inline openpype/hosts/flame/plugins/publish/extract_subset_resources.py:187]:
```suggestion
        version_frame_start = frame_start
        if include_handles:
            version_frame_start = frame_start_handle

        if r_speed != 1.0:
            if retimed_handles:
                instance.data["versionData"].update({
                    "frameStart": version_frame_start,
                    "frameEnd": (
                        (version_frame_start + source_duration_handles - 1)
                        - (r_handle_start + r_handle_end)
                    )
                })
            else:
                instance.data["versionData"].update({
                    "handleStart": handle_start,
                    "handleEnd": handle_end,
                    "frameStart": version_frame_start,
                    "frameEnd": (
                        (version_frame_start + source_duration_handles - 1)
                        - (handle_start + handle_end)
                    )
                })
```

Comment (jakubjezek001) [reply openpype/hosts/flame/plugins/publish/extract_subset_resources.py]: thanks @iLLiCiTiT ;)

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**